Examples/Detectors/Geant4Detector/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_library(
+  ActsExamplesDetectorGeant4 SHARED
+  src/Geant4Detector.cpp
+  src/Geant4DetectorService.cpp)
+target_include_directories(
+    ActsExamplesDetectorGeant4
+  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
+target_link_libraries(
+    ActsExamplesDetectorGeant4
+  PUBLIC ActsCore ActsExamplesFramework ActsExamplesGeant4 ActsPluginGeant4)
+
+install(
+  TARGETS ActsExamplesDetectorGeant4
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
nitpick: empty line at the end of the file

Reply:
And fixed.